inspektor/lint.py

```diff
@@ -63,8 +69,9 @@ def get_opts(self):
                 pylint_args.append('--disable=%s' % self.ignored_errors)
             if self.enabled_errors:
                 pylint_args.append('--enable=%s' % self.enabled_errors)
-            pylint_args.append('--reports=no')
-            if sys.version_info[:2] > (2, 6):
+            if self._pylint_has_option('--reports'):
```

Comment:
Yep, this should be safest. I'd prefer if we check `--reports=<y` to avoid possible clashes with `--reports-whatever` or even just example `--reports=n` which might be listed in some comment, but that is probably just paranoiac...